src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js

```diff
@@ -118,13 +124,13 @@ export const WidgetEventEditPlain = ({
                     <div className={classes.menuActions}>
                         <ConditionalTooltip
                             content={i18n.t('You don\'t have access to edit this event')}
```

Comment:
In the [design documentation](https://docs.google.com/document/d/1xPXZ9sBizX8Yv9J2CO2G5hFRPrVzoNMO71uRiNVug5I/edit#heading=h.7vovce51x8b1) I suggested "Editing this event is disabled because it is complete” but I think I like your suggestion more @simonadomnisoru :)

Reply:
I'll implement your suggestion, @simonadomnisoru. However, I believe we should also retain the original one. The button will be disabled not only when the _block entry form_ setting is active for the program stage, but also if the user lacks _write access_. This means the button could be disabled even if the _block entry form_ setting isn't in effect. What do you think about something like this @karolinelien and @simonadomnisoru? Thanks!
`const tooltipContent = blockEntryForm ?
        i18n.t('The event cannot be edited after it has been completed') :
        i18n.t('You don\'t have access to edit this event');`

Reply:
I'll implement your suggestion, @simonadomnisoru. However, I believe we should also retain the original one. The button will be disabled not only when the block entry form setting is active for the program stage, but also if the user lacks write access. This means the button could be disabled when only write access is missing, even if the block entry form setting isn't in effect. What do you think about something like this @karolinelien and @simonadomnisoru? Thanks!

```
const tooltipContent = blockEntryForm ?
        i18n.t('The event cannot be edited after it has been completed') :
        i18n.t('You don\'t have access to edit this event');
```